.gitignore

```diff
@@ -41,6 +41,7 @@ mixxx-test
 
 src/test/test_data
 src/test/golden_buffers/*.actual
+*test.bcp.xml
```

Comment:
Is this really needed? I thought we now write them to `/tmp`?

Reply:
Still added them to gitignore as a fallback.

Reply:
Not necessarily, but theoretically there could be the case that the temp dir is unwritable and we fall back to the current dir. Then it is nice to have them ignored anyways.

Reply:
Where is the fallback code? I think in that case the test should just fail and not handle this gracefully.

Reply:
> It is very important to test that the temporary directory could be created, using isValid().
https://doc.qt.io/qt-5/qtemporarydir.html#details

But yeah, I'll just assert that it's valid.